City-of-Helsinki/helsinki-paatokset
# PP-310: Accessibility + typography fixes

Accessibility + typography fixes for the news integration.

To test:
- Run `drush cim -y; drush cr`.
- Run the migration: `drush mim paatokset_news --update` (update flag is important)
- Add 'news liftups' paragraph to your frontpage if you don't have one already

Styles to check on frontpage:
- Main liftup title has normal h3 font size
- Other liftups have h4 appearence, but are h3 elements
- Main liftup image has alt and title attributes

Navigate to a news article page, preferably one with an image. Check there that: 
- Article image has alt and title attributes
- If there are any articles without image in the sidebar on the right the placeholder is diplayed correctly (without an extra eye in the cottage - bad example below)
![image](https://user-images.githubusercontent.com/49063836/157215700-f256f23e-8d3e-4262-9951-014f97bf53fc.png)
